avm/res/cache/redis-enterprise/tests/e2e/max/main.test.bicep

```diff
@@ -81,10 +79,6 @@ module testDeployment '../../../main.bicep' = [
             name: 'RediSearch'
           }
         ]
-        persistence: {
-          type: 'aof'
-          frequency: '1s'
-        }
```

Comment:
Just wondering why this was removed?

Reply:
Hey @JeffreyCA,
I did because I got an error that was pretty explicit about not being able to do this for some combination of properties. Let me try and find the failed run.